packages/core/admin/admin/src/content-manager/components/InputJSON/index.js

```diff
@@ -23,6 +16,26 @@ const WAIT = 600;
 const stringify = JSON.stringify;
 const DEFAULT_THEME = 'solarized dark';
 
+const loadCss = async () => {
```

Comment:
WYT of lazy loading the JSON input itself instead of all the associated styles and behaviour? I'm feeling that there might be a flicker on a low end connection showing the inputjson in a raw state and then loading the extra stuff